.circleci/soltest_all.sh

```diff
@@ -28,7 +28,7 @@ set -e
 
 REPODIR="$(realpath "$(dirname "$0")"/..)"
 
-EVM_VALUES=(homestead byzantium constantinople petersburg istanbul)
+EVM_VALUES=(homestead byzantium constantinople petersburg istanbul berlin)
```

Comment:
I think we should have a variable for the default EVM here and use it instead of the hard-coded value in the code below.
```suggestion
EVM_VALUES=(homestead byzantium constantinople petersburg istanbul berlin)
DEFAULT_EVM=berlin
[[ " ${EVM_VALUES[@]} " =~ " $DEFAULT_EVM " ]]
```

Reply:
I added a variable for default. Also where is `[[ " ${EVM_VALUES[@]} " =~ " $DEFAULT_EVM " ]]` supposed to go?

Reply:
Just where it is in my snippet above.
It's an "assert" that will return false (and end the script due to `set -e`) if `$EVM_VALUE` is not one of the values in `$EVM_VALUES`.

Reply:
While somehow I managed to understand what it supposed to just by looking at, perhaps it is better to put a comment since at least @hrkrshnn asked what is it doing.